<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableWidget{
color: #E5E5E5;
border: #FFE5B4;
border-radius: 1px;
font-family: 'Roboto', sans-serif;
font-size: 16px; /* Tamanho do texto */
line-height: 1.5; /* Altura da linha para melhor legibilidade */
background: black;
}

QLabel{
color: #E5E5E5;
border: #FFE5B4;
border-radius: 1px;
font-family: 'Roboto', sans-serif;
font-size: 16px; /* Tamanho do texto */
line-height: 1.5; /* Altura da linha para melhor legibilidade */
}
QLabel#label_title{
color: #E5E5E5;
border: #FFE5B4;
border-radius: 1px;
font-family: 'Roboto', sans-serif;
font-size: 30px; /* Tamanho do texto */
line-height: 1.5; /* Altura da linha para melhor legibilidade */
}

QLineEdit{
background:Silver;
border: 2px rgb(85, 170, 255) solid;
border-radius: 10px;
color: #B87333
}
QLineEdit:focus {
Background: #F4FDFF;
border: 2px  solid #FFE5B4;
color: #B87333
}

QComboBox{
background:Silver;
border: 2px rgb(85, 170, 255) solid;
border-radius: 10px;
color: #B87333
}

QPushButton{
background:#808B8E;
border: 2px rgb(85, 170, 255) solid;
border-radius: 10px;
color: #E5E5E5
}

QPushButton:hover{
background:#43d7f7;
border: 2px rgb(85, 170, 255) solid;
border-radius: 10px;
color: black
}

QPushButton#btn_logout:hover{
background:#FF4C4C;
border: 2px rgb(85, 170, 255) solid;
border-radius: 10px;
color: black
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Roboto,sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>TechBooks</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_user">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>user</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>book-reader.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_books">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>781</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Adicionar um novo Livro</string>
    </property>
    <widget class="QPushButton" name="btn_add_book">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Adicionar Livro</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_user_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Título</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_user_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Autor</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_titulo">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_autor">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_ano">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_user_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ano de Lançamento</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_user_5">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Link da imagem</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_user_6">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ISBN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_isbn">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_paginas">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_foto">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>60</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_user_7">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>100</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Número de páginas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
